pkgs/development/libraries/smpeg2/default.nix

```diff
@@ -24,12 +24,16 @@ stdenv.mkDerivation rec {
   buildInputs = [ SDL2 ]
     ++ lib.optional stdenv.isDarwin darwin.libobjc;
 
+  outputs = [ "out" "dev" "man" ];
+
   preConfigure = ''
     sh autogen.sh
   '';
 
   postInstall = ''
-    wrapProgram $out/bin/smpeg2-config \
+    mkdir -p $dev/bin
+    mv $out/bin/smpeg2-config $dev/bin/smpeg2-config
```

Comment:
Those two lines (mkdir+mv) can be replaced with:

```
moveToOutput bin/smpeg2-config "$dev"
```

Reply:
That's a nice helper! TIL. Used it as is.